invenio_drafts_resources/services/records/tasks.py

```diff
@@ -0,0 +1,11 @@
+from celery import shared_task
+from .api import Draft, ParentRecord, Record
+
+@shared_task(ignore_result=True)
+async def cleanup_drafts(seconds=3600):
+    """Hard delete of soft deleted drafts.
```

Comment:
Line-length wise it will be something like this:

```
    :param int seconds_param: numbers of seconds that should pass since the 
        soft-deletion of a draft in order to be hard deleted.
    """
```


Also, if you dont' have it already, then configure your editor to show a line at 79 chars. I e.g. have the following in  VS Code: 

```
    "editor.rulers": [
        79
    ],
    "[git-commit]": {
        "editor.rulers": [
            50, 72
        ]
    },
```